source/taup/taup_time.rst

```diff
@@ -0,0 +1,159 @@
+
+输出共 9 列，从左到右分别为：
+
+1.  震中距（度）：地震震中与台站的大圆弧路径长度
+2.  震源深度（km）
+3.  震相名
+4.  震相走时（秒）
+5.  射线参数（秒/度）
+
+    .. note::
+
+        射线参数有三种常用单位：秒/度（s/deg ）、秒/弧度（s/radian）、秒/千米（s/km）。
+        三者之间的关系为（:math:`R_0` 为地球半径，假设为 6371 km）：
+
+        .. math::
+
+            1 s/deg = \frac{1}{\pi*R_0/180} s/km = 1/111.19492 s/km
+
+        .. math::
+
+            1 s/deg = \frac{1}{\pi/180} s/radian = 57.29578 s/radian
+
+6.  出射角（度）：即射线从震源出射时与\ **垂直向下**\ 方向的夹角，取值范围为 0 到 180°。
+
+    - 0° 到 90° 表示射线向下出射（例如 P 震相）
+    - 90° 到 180° 表示射线向上出射（例如 pP 震相)
+
+    .. note::
+
+       不同文章对出射角的定义不同，使用时应格外注意。
+
+       老版本的 TauP 对出射角的定义不同，可视为 BUG，应避免使用老版本 TauP。
+
+7.  入射角（度）：射线入射到台站时与\ **垂直向上**\ 方向的夹角。
+8.  Purist 距离：地震波实际走过的圆弧距离。
+
+    该值可能与震中距不同，例如 50° 震中距处记录到的 PKKKP 震相，实际走过的
+    圆弧距离为 410°::
+
+        $ taup_time -deg 50 -ph PKKKP
+
+        Model: iasp91
+        Distance   Depth   Phase   Travel    Ray Param  Takeoff  Incident  Purist    Purist
+          (deg)     (km)   Name    Time (s)  p (s/deg)   (deg)    (deg)   Distance   Name
+        -----------------------------------------------------------------------------------
+          50.00     0.0   PKKKP   2519.56     2.242      6.72     6.72   410.00   = PKKKP
+
+9.  Purist 震相名：地震波的“真实”震相名。
+
+    例如，若想要计算 SS 震相在 660 km 不连续面的下表面的反射波走时，可指定其震相名为
+    ``S^660S``。由于 PREM 模型中 660 km 不连续面的“真实”深度为670 km，此时
+    TauP 实际返回的时从 670 km 处反射回的震相，此时 Purist 震相名会显示为 ``S^670S``::
+
+        $ taup_time -mod prem -deg 120 -ph S^660S
+
+        Model: prem
+        Distance   Depth   Phase    Travel    Ray Param  Takeoff  Incident  Purist    Purist
+          (deg)     (km)   Name     Time (s)  p (s/deg)   (deg)    (deg)   Distance   Name
+        ------------------------------------------------------------------------------------
+          120.00     0.0   S^660S   1975.75    12.336     20.79    20.79   120.00   * S^670S
+
+当使用 ``--rel`` 选项时，``taup_time`` 可输出相对于某个指定震相的走时差。例如，
+下面的命令中，最后一列输出了所有震相相对于 PcP 震相的走时差::
+
+    $ taup_time -ph P,S,PcP,ScS,PKiKP -deg 60 -h 300 -mod prem --rel PcP
+
+    Model: prem
+    Distance   Depth   Phase   Travel    Ray Param  Takeoff  Incident  Purist    Purist Relative to
+      (deg)     (km)   Name    Time (s)  p (s/deg)   (deg)    (deg)   Distance   Name       PcP
+    -----------------------------------------------------------------------------------------------
+       60.00   300.0   P        574.72     6.739     33.65    20.58    60.00   = P      -42.32
+       60.00   300.0   PcP      617.05     4.016     19.28    12.09    60.00   = PcP      0.00
+       60.00   300.0   PKiKP    993.45     1.245      5.88     3.72    60.00   = PKiKP  376.41
+       60.00   300.0   S       1043.98    12.659     34.16    21.37    60.00   = S      426.93
+       60.00   300.0   ScS     1134.89     7.480     19.38    12.43    60.00   = ScS    517.85
+
+信息提取
+--------
+
+上面介绍的输出中包含了很多信息，想要在脚本中提取出想要的信息有些麻烦。所以
+``taup_time`` 提供了一些选项，使得在脚本中计算走时什么的更加方便。
+
+走时
+~~~~
```

Comment:
I usually use ++++ to indicate the third level, so let's make it a rule to use ~~~ now?

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
Should we follow this?

Reply:
# with overline, for parts
* with overline, for chapters

What do these two mean?

We may only need the following three terms:
=, for sections
-, for subsections
^, for subsubsections

Reply:
> What do these two mean?
> 
> * "# with overline", for parts
> * "* with overline", for chapters

They're just higher level headings.



> We may only need the following three terms:
> =, for sections
> -, for subsections
> ^, for subsubsections

I think yes.